public/app/ml-worker-fetch-cache.js

```
@@ -389,81 +389,118 @@ window._vipCacheStatus = async function cacheStatus() {
 
+const MODEL_DISPLAY_NAMES = {
+  silero_vad:      'Voice Activity Detector',
+  silero_vad_int8: 'Voice Activity Detector (fast)',
+  rnnoise:         'Noise Suppressor',
+  demucs_v4:       'Demucs Voice Separator',
+  bsrnn_vocals:    'Band-Split RNN Vocals',
+};
+
 function _ensureProgressPanel(keys) {
+  // Create backdrop if not present
+  let backdrop = document.getElementById('vip-model-load-backdrop');
```

ESLint

[error] 'document' is not defined.

(no-undef)